R/sendmailR.r

```diff
@@ -218,7 +270,17 @@ sendmail <- function(from, to, subject, msg, cc, bcc, ...,
     server <- get_value("smtpServer", "localhost")
     port <- get_value("smtpPort", 25)
```

Comment:
When using `curl`, should we switch to port 587 because we are more than likely submitting mail to a smart host that will forward it to the final destination on our behalf? 

I would change this globally, but it would be a breaking change and I don't want to break peoples workflows.

Reply:
Only curl will default to 587. Port 25 will be the default for the internal engine.